src/components/client/core/Banner/Banner.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+import { FiX as CloseIcon } from "react-icons/fi";
+
+import Button from "components/client/core/Button/Button";
+import Icon from "components/client/core/Icon/Icon";
+import { panda } from "generated/panda/jsx";
+import { banner } from "generated/panda/recipes";
+
+import type { BannerVariantProps } from "generated/panda/recipes";
+import type { ComponentProps } from "react";
+
+export interface Props
+  extends Omit<ComponentProps<typeof panda.div>, "color">,
+    BannerVariantProps {
+  closable?: boolean;
+}
```

Comment:
Can simplify to:

```ts
export interface Props
  extends ComponentProps<typeof panda.div>,
    BannerVariantProps {
  closable?: boolean;
}
```

I know this was done when we were discussing `color` variants, but that is no long an issue (no prop collision) with your updated recipe!

Reply:
Good eye.